app/views/surveys/show.html.erb

```diff
@@ -13,25 +13,29 @@
     <% end %>
   </ul>
 
-  <!-- Tab panes -->
   <div class="tab-content container">
     <div id="home" class="container tab-pane active"><br>
       <h2>Survey Preview</h2>
       <div class= "text-left survey-box container shadow p-3 mb-5 bg-white rounded">
         <% @survey.questions.each_with_index do |question , index| %>
-        <br>
-          <p class="font-weight-bold">Question <%= "#{index+1} : #{question.statement}" %></p>
-          <% question.options.each_with_index do |option, option_index| %>
+          <br>
+          <% if question.required %>
+            <p class="font-weight-bold">Question <%= "#{index+1} : #{question.statement} *" %></p>
```

Comment:
you just need * sign then write condition on it only like 
<p class="font-weight-bold">Question <%= "#{index+1} : #{question.statement} #{'*' if question.required? }" %></p>